packages/react-vapor/src/components/browserPreview/tests/BrowserPreview.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import {mount, ReactWrapper, shallow} from 'enzyme';
+import * as React from 'react';
+
+import {Tooltip} from '../../tooltip';
+import {BrowserPreview, BrowserPreviewProps} from '../BrowserPreview';
+
+describe('BrowserPreview', () => {
+    let component: ReactWrapper<BrowserPreviewProps>;
+    const mountWithProps = (props: BrowserPreviewProps) => {
+        component = mount(<BrowserPreview {...props} />, {
+            attachTo: document.getElementById('App'),
+        });
```

Comment:
You don't need to attach the component to the DOM, it will be faster:
```suggestion
        component = mount(<BrowserPreview {...props} />);
```